doc/d-practical-exercises/3-invkine.md

```diff
@@ -21,28 +21,35 @@ the tool. They are displayed when the robot moves.
 Example of how to use the robot is has below.
 
 ```py
-from os.path import join
-import pinocchio as se3
-from mobilerobot import MobileRobotWrapper
+import pinocchio as pin
 from pinocchio.utils import *
-
-PKG = '/opt/openrobots/share'
-URDF = join(PKG, 'ur5_description/urdf/ur5_gripper.urdf')
-
-robot = MobileRobotWrapper(URDF, [PKG])
+from os.path import dirname, join, abspath
+from mobilerobot import MobileRobotWrapper
```

Comment:
I thought the purpose of this PR was to remove mobilerobot? :)
Could you instead use robotwrapper here?